### PR TITLE
[BPK-4141] Migrate Android pages to use AndroidComponentPage

### PR DESCRIPTION
**Pull request by shaundon**
Also migrated web accordion and alignment pages. Basically all pages beginning in `A`.

### CODE REVIEW

File: docs/src/components/ComponentPage/AndroidComponentPage.js [hunk 1]
```diff
@@ -29,10 +29,11 @@ import ComponentScreenshots from '../DocsPageBuilder/ComponentScreenshots';
 import ComponentPage from './ComponentPage';
 
 type Props = {
-  documentationId: string,
+  documentationId: ?string,
```
Comment (shaundon) [inline docs/src/components/ComponentPage/AndroidComponentPage.js:32]: Turns out my assumption from the last PR was wrong. There is one Android component that doesn't have an accompanying documentation link — icons.

File: docs/src/components/ComponentPage/AndroidComponentPage.js [hunk 2]
```diff
@@ -29,10 +29,11 @@ import ComponentScreenshots from '../DocsPageBuilder/ComponentScreenshots';
...
   readme: string,
   screenshots: [
     {
+      blurb: ?string,
```
Comment (shaundon) [inline docs/src/components/ComponentPage/AndroidComponentPage.js:36]: I missed this last time around, so any screenshots with accompanying blurbs weren't displayed. Fixed now (also made the same change to iOS and RN).

File: docs/src/components/ComponentPage/WebComponentPage.js [hunk 3]
```diff
@@ -36,7 +36,7 @@ type Props = {
     blurb: Node,
     examples: Node,
   }>,
-  readme: string,
+  readme: ?string,
```
Comment (shaundon) [inline docs/src/components/ComponentPage/WebComponentPage.js:39]: The alignment page doesn't have a readme, so I had to make it optional for web components.

**Review by olliecurtis — COMMENTED** [review]
LGTM - Just going to run it locally

**Review by georgegillams — APPROVED** [review]

**Pull request merged**